caravel/templates/caravel/basic.html

```diff
@@ -21,7 +21,7 @@
 
   <body>
     {% block navbar %}
-      {% if not viz or viz.request.args.get("standalone") != "true" %}
+      {% if not dash_standalone and (not viz or viz.request.args.get("standalone") != "true") %}
```

Comment:
If we change it to 'or', when dash_standalone=true, not dash_standalone = false, but viz is not passed in, so not viz will be true, which makes the if statement to be true and header is displayed.